platform/solr/solr-dependencies/pom.xml

```diff
@@ -21,199 +21,627 @@
     </parent>
     <artifactId>solr-dependencies</artifactId>
     <name>DDF :: Platform :: Solr :: Dependencies</name>
-    <description>Shaded jar of Solr core dependencies.</description>
-    <packaging>jar</packaging>
-    <properties>
-        <jts.version>1.15.0</jts.version>
-        <dropwizard.version>3.2.6</dropwizard.version>
-        <s2.version>1.0.0</s2.version>
-    </properties>
+    <description>Contains solr dependencies. Exports solr related packages required by DDF.</description>
+    <packaging>bundle</packaging>
     <dependencies>
         <dependency>
             <groupId>org.apache.solr</groupId>
             <artifactId>solr-core</artifactId>
-            <exclusions>
-                <exclusion>
-                    <artifactId>rrd4j</artifactId>
-                    <groupId>org.rrd4j</groupId>
-                </exclusion>
-                <exclusion>
-                    <artifactId>log4j-1.2-api</artifactId>
-                    <groupId>org.apache.logging.log4j</groupId>
-                </exclusion>
-                <exclusion>
-                    <artifactId>log4j-api</artifactId>
-                    <groupId>org.apache.logging.log4j</groupId>
-                </exclusion>
-                <exclusion>
-                    <artifactId>log4j-core</artifactId>
-                    <groupId>org.apache.logging.log4j</groupId>
-                </exclusion>
-                <exclusion>
-                    <artifactId>log4j-slf4j-impl</artifactId>
-                    <groupId>org.apache.logging.log4j</groupId>
-                </exclusion>
-            </exclusions>
+            <version>7.4.0</version>
```

Comment:
Using the solr version property would be good. I don't think we gain anything by making properties for the other hardcoded versions of solrs dependencies though since they won't be shared outside of this pom.

Reply:
I'm with @figliold here. We should never hardcode versions in any other poms than the top one. Having all versions in one place makes it easier to see what we are using as opposed to digging through all poms. We don't know what will happen in the future and doing this is how we suddenly see different versions of stuff appearing.

Reply:
this is why we can actually add comments where these gets defined.